modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector-reference.adoc

```diff
@@ -3,283 +3,338 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+Version 2.3.1
+
 == Configurations
 ---
 [[dynamics-config]]
-== Dynamics 365
+=== Dynamics 365
+
+
+==== Parameters
 
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 |Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
-| Connection a| * <<dynamics-config_connection, Connection>>
-* <<dynamics-config_oauth-user-pass, OAuth Username Password>>
+| Connection a| * <<dynamics-config_connection, Connection>> 
```

Comment:
Thought there was an extraneous asterisk before the angle brackets, but displays fine. So, if incorrect, please ignore this comment.

Reply:
An asterisk followed by a space is a bullet list item -- it's correct.